packages/viz/src/lib/Layer.ts

```diff
@@ -90,18 +98,18 @@ export class Layer {
     const metadata = this._source.getMetadata();
 
     const styleProps = this._style
-      ? this._style.getProperties(this._source)
+      ? this._style.getProperties(this)
       : undefined;
 
     // Get properties of the layer
     const props = this._source.getProps();
```

Comment:
should rename that method in the source, to have `const props = this._source.getLayerProps();` ?